Support for pddl4j (Dorian and Ronan's work)

dodo64 opened the pull request:
Closes #145

Illurian committed:
Ajout bouton plug-in et PDDL4T

Commit:
premier commit : ati et traduction pddl

Illurian committed:
Modification de la barre de menu

Appel du programme PDDL4T afin d'afficher les options de traitement de
PDDL4T

maelvls committed:
latin1 to utf8

maelvls committed:
fixed wrong import to com.sun.glass.events

Commit:
avant push

maelvalais commented:
Coucou,
Je vois que `travis` et `appveyor` ne sont pas contents. J'ai remarqué que `javac` donne une erreur : 

    /home/travis/build/touist/touist/touist-gui/src/ATIGraph.java:4: error: package fr.uga.pddl4j.encoding does not exist

Commit:
librairie pddl4j

maelvalais commented:
Possible de mettre `pddl4j.jar` dans `libs` avec les autres .jar ? Ça serait nickel

maelvalais commented:
Ah oui, il faut aussi modifier `build.xml` pour qu'il prenne en compte le nouveau .jar pendant le build. Je vais m'en occuper.

maelvls committed:
Added pddl4j.jar to build.xml to make it work on `ant build`

Commit:
relocate

Commit:
application des fonctions de traductions et d'instanciations à l'interface

Commit:
ajout package

maelvalais commented:
- `package pddl4t;` doit être en tout début de fichier (regarder erreur travis-ci)
- les fichiers du package pddl4t devraient plutôt être dans le dossier src/pddl4t 
- apparemment `ATIGraph` n'a pas été déclarée public

Illurian committed:
Ajout des action pour prendre en compte les choix des jRadioButton

Vérifie les JRadioButton pour apeller seulement les programmes
correspondants.

Commit:
(y)

maelvalais commented on touist-gui/src/ATIGraph.java:
```diff
@@ -3,7 +3,7 @@
 import fr.uga.pddl4j.encoding.CodedProblem ;
 import fr.uga.pddl4j.util.* ;
-package pdd4t;
+
```
Tu as enlevé `package pdd4t` ?

maelvls committed:
Moved pddl4t files to a proper package

maelvls committed:
Import pddl4t package to be able to see `ATIGraph` (among others)

maelvls committed:
Fixed name difference between class name and file name

maelvls committed:
Set ATIGraph and Traduction classes in public (because they are now in packages)

Commit:
ok

Commit:
ça build

Illurian committed:
Commit All

bientot fini

Illurian committed:
Commit presqfin

maelvalais commented:
Quelqu'un peut-il réparer le build ? Merci :)

Illurian committed:
Commit final!

Illurian committed:
Last maj

maelvls committed:
ISO-8859-1 to UTF-8 (second time...)

What I typed:

    iconv -f ISO-8859-1 -t UTF-8 touist-gui/src/gui/menu/PDDL4T_interface.java | sponge touist-gui/src/gui/menu/PDDL4T_interface.java

maelvalais commented:
Le fichier `PDDL4T_interface.java` était de nouveau revenu en encodage `ISO-5589-1` (latin1). Faites gaffe :-/

maelvalais commented:
Je viens de rebase le PR pour
- intégrer le PR #151 directement dans ce PR
- linéariser l'historique qui était un peu illisible à cause des multiples "branches internes" au PR
- virer les merge commits inutiles
